polymer.json

```diff
@@ -2,9 +2,29 @@
   "entrypoint": "index.html",
   "shell": "src/lancie-admin-app.html",
   "sourceGlobs": [
-   "images-optimized/**/*"
+   "images-optimized/**/*",
+   "scripts/*"
   ],
   "fragments": [
+    "src/lancie-home-page/lancie-home-page.html",
+    "src/lancie-my-area/lancie-my-area.html",
+    "src/lancie-my-area/lancie-accept-ticket-transfer.html",
+    "src/lancie-my-area/lancie-success-ticket-transfer.html",
+    "src/lancie-ticket-page/lancie-ticket-page.html",
+    "src/lancie-seatmap/lancie-seatmap.html",
+    "src/lancie-rfid/lancie-rfid.html",
+    "src/lancie-ticket-page/lancie-ticket-profile-intercept.html",
+    "src/lancie-login/lancie-register-confirmation.html",
+    "src/lancie-login/lancie-check-email.html",
+    "src/lancie-contact/lancie-contact.html",
+    "src/lancie-contact/lancie-contact-success.html",
+    "src/lancie-order-check.html",
+    "src/lancie-404.html",
+    "src/lancie-construction.html",
+    "src/lancie-password/lancie-password-reset.html",
+    "src/lancie-password/lancie-password-reset-success.html",
+    "src/lancie-password/lancie-request-password-reset.html",
+    "src/lancie-password/lancie-request-password-reset-success.html"
```

Comment:
All these pages can be removed as of now, we do not require these for lazy-loading.